configuration.c

```diff
@@ -143,9 +143,18 @@ enum audio_driver_enum
    AUDIO_NULL
 };
 
+enum microphone_driver_enum
+{
+   MICROPHONE_ALSA = AUDIO_NULL + 1,
+   MICROPHONE_ALSATHREAD,
+   MICROPHONE_SDL2,
+   MICROPHONE_WASAPI,
+   MICROPHONE_NULL
+};
+
 enum audio_resampler_driver_enum
 {
-   AUDIO_RESAMPLER_CC       = AUDIO_NULL + 1,
+   AUDIO_RESAMPLER_CC       = MICROPHONE_NULL + 1,
```

Comment:
Is it a mistake to replace AUDIO_NULL with MICROPHONE_NULL here?

Reply:
Nope. Each driver is given a distinct number, right? The value of the first constant in each `enum` here is equal to the value of the _last_ constant in the previous `enum`.